Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml

```
@@ -38,6 +44,8 @@ configuration:
   - exchangeserver
   required: false
   type: 16
+  section: Collect
+  advanced: true
```

```suggestion
  section: Collect
```
I think it will not fetch anything if this is empty

---

Its not required and the code seems to handle empty case..